src/pages/saksbehandling/beregning/VisBeregning.tsx

```diff
@@ -15,8 +17,32 @@ import styles from './visBeregning.module.less';
 
 interface Props {
     beregning: Beregning;
+    forventetinntekt: number;
 }
 
+const fradragMedForventetinntekt = (
+    fragdragsArray: Array<Fradrag>,
+    forventetinntekt: number,
+    intl: IntlShape
+): Array<Fradrag> => {
+    const fradragPartioned = pipe(
+        fragdragsArray,
+        arr.partition((f) => f.type === Fradragstype.Arbeidsinntekt)
+    );
+
+    if (fradragPartioned.right.reduce((acc, fradrag) => acc + fradrag.beløp, 0) >= forventetinntekt) {
+        return fragdragsArray;
+    }
+
+    fradragPartioned.left.push({
+        type: ForventetInntektfradrag,
+        beløp: forventetinntekt,
+        beskrivelse: intl.formatMessage({ id: 'display.brukerForventetinntekt' }),
+    });
+
+    return fradragPartioned.left;
+};
```

Comment:
Typo i `fragdragsArray`; kan IMO forenkle det til `fradrag`. Pluss at vi kan gi `left` og `right` andre navn så de blir litt lettere å skjønne :)

```suggestion
const fradragMedForventetinntekt = (
    fradrag: Array<Fradrag>,
    forventetinntekt: number,
    intl: IntlShape
): Array<Fradrag> => {
    const { left: andreFradrag, right: arbeidsinntektfradrag } = pipe(
        fradrag,
        arr.partition((f) => f.type === Fradragstype.Arbeidsinntekt)
    );

    if (arbeidsinntektfradrag.reduce((acc, f) => acc + f.beløp, 0) >= forventetinntekt) {
        return fradrag;
    }

    return [
        ...andreFradrag,
        {
            type: ForventetInntektfradrag,
            beløp: forventetinntekt,
            beskrivelse: intl.formatMessage({ id: 'display.brukerForventetinntekt' }),
        }
    ];
};
```

Reply:
ah, så det var sånn det gjøres! :)